sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -191,9 +193,25 @@ def _create_pipeline_workflow(self, args, pipeline, op_transformers=None, pipeli
       if pipeline_conf.timeout:
         pipelinerun['spec']['timeout'] = '%ds' % pipeline_conf.timeout
 
-
       workflow = workflow + [pipelinerun]
 
+    # Use regex to replace all the Argo variables to Tekton variables.
+    # For variables that are unique to Argo, we prompt a warning to warn users on the non-supported variables.
```

Comment:
can we list which specific variables we cannot support?

Reply:
I can add a link to the issue/doc since the variables will change over time.